src/features/Instructors/InstructorsTable/columns.jsx

```diff
@@ -1,26 +1,38 @@
+/* eslint-disable react/prop-types, no-nested-ternary */
+import moment from 'moment';
+
 const columns = [
   {
-    Header: 'User Name',
+    Header: 'Instructor',
     accessor: 'instructorUsername',
   },
   {
-    Header: 'Name',
-    accessor: 'instructorName',
+    Header: 'Last seen',
+    accessor: 'lastAccess',
+    Cell: ({ row }) => {
+      const currentDate = moment(Date.now());
+      const lastDate = moment(new Date(row.values.lastAccess));
+      const diffHours = currentDate.diff(lastDate, 'hours');
+      const diffDays = currentDate.diff(lastDate, 'days');
+      const diffWeeks = currentDate.diff(lastDate, 'weeks');
+      return (
+        <span>{diffHours < 24
+          ? 'Today'
+          : diffDays < 7
+            ? `${diffDays} days ago`
+            : `${diffWeeks} weeks ago`}
```

Comment:
```suggestion
            : `${diffWeeks} wks ago`}
```
Following the figma:
![image](https://github.com/Pearson-Advance/frontend-app-institution-portal/assets/30726391/b842d833-bf71-4511-b9fc-fe1f7e9e06a0)